client/src/App.js

```diff
@@ -16,14 +16,33 @@ function App() {
 
   const handleChange = (event, newValue) => {
     setValue(newValue);
+    funclick();
+  };
+
+
+  const funclick = function (){
+  const baseUrl = window.location.origin;
+  const tabs = document.querySelector('.MuiTabs-flexContainer'); 
+  const state = history.state;
+  tabs.addEventListener('click', tab => 
+    {
+      if (tab.target) 
+      {
+        const links = tab.target.getAttribute('link');
+        const newUrl = new URL(links, baseUrl);
```

Comment:
before that need to check if link attribute exist. if not, it means we don't have this page on the website

Reply:
сonst links = tab.target.getAttribute('link');
if (links === '') {
  return; // exit function because we can not send user to any page
}
const newUrl = new URL(links, baseUrl);